diff -Nur anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/root_password.py anaconda-29.24.7/pyanaconda/ui/gui/spokes/root_password.py
--- anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/root_password.py	2019-01-30 20:34:33.200096820 -0500
+++ anaconda-29.24.7/pyanaconda/ui/gui/spokes/root_password.py	2019-01-31 02:41:53.530096820 -0500
@@ -274,7 +274,7 @@
                     self.needs_waiver = False
             else:
                 if not self._validity_check.result.success:
-                    self.can_go_back = True
+                    self.can_go_back = False
                     self.needs_waiver = True
                 elif not self._ascii_check.result.success:
                     self.can_go_back = True
diff -Nur anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/user.py anaconda-29.24.7/pyanaconda/ui/gui/spokes/user.py
--- anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/user.py	2019-01-30 20:34:33.200096820 -0500
+++ anaconda-29.24.7/pyanaconda/ui/gui/spokes/user.py	2019-01-31 02:42:17.751096820 -0500
@@ -640,7 +640,7 @@
                 if not self._confirm_check.result.success:
                     self.can_go_back = False
                 if not self._validity_check.result.success:
-                    self.can_go_back = True
+                    self.can_go_back = False
                     self.needs_waiver = True
                 elif not self._ascii_check.result.success:
                     self.can_go_back = True
