diff -Naur anaconda-21.48.22.134-old/data/interactive-defaults.ks anaconda-21.48.22.134/data/interactive-defaults.ks
--- anaconda-21.48.22.134-old/data/interactive-defaults.ks	2018-11-03 09:06:09.893000000 -0400
+++ anaconda-21.48.22.134/data/interactive-defaults.ks	2018-11-03 09:07:25.449000000 -0400
@@ -5,9 +5,9 @@
 
 %anaconda
 # Default password policies
-pwpolicy root --notstrict --minlen=6 --minquality=1 --nochanges --notempty
-pwpolicy user --notstrict --minlen=6 --minquality=1 --nochanges --emptyok
-pwpolicy luks --notstrict --minlen=6 --minquality=1 --nochanges --notempty
+pwpolicy root --notstrict --minlen=8 --minquality=1 --nochanges --notempty
+pwpolicy user --notstrict --minlen=8 --minquality=1 --nochanges --emptyok
+pwpolicy luks --notstrict --minlen=8 --minquality=1 --nochanges --notempty
 # NOTE: This applies only to *fully* interactive installations, partial kickstart
 #       installations use defaults specified in pyanaconda/pwpolicy.py.
 #       Automated kickstart installs simply ignore the password policy as the policy
