From 99970253a5600555118a64046259b7ef638655df Mon Sep 17 00:00:00 2001
From: t_feng <fengtao40@huawei.com>
Date: Fri, 11 Sep 2020 11:26:41 +0800
Subject: [PATCH] add boot args for smmu and video

---
 pyanaconda/modules/storage/bootloader/grub2.py | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/pyanaconda/modules/storage/bootloader/grub2.py b/pyanaconda/modules/storage/bootloader/grub2.py
index 17e46e5..a70ba7a 100644
--- a/pyanaconda/modules/storage/bootloader/grub2.py
+++ b/pyanaconda/modules/storage/bootloader/grub2.py
@@ -271,6 +271,14 @@ class GRUB2(BootLoader):
         if blivet.arch.is_aarch64():
             log.info("check boot args:%s", arg_str)
             arg_str += " crashkernel=1024M,high"
+            if "smmu.bypassdev=0x1000:0x17" not in arg_str:
+                arg_str += " smmu.bypassdev=0x1000:0x17"
+            if "smmu.bypassdev=0x1000:0x15" not in arg_str:
+                arg_str += " smmu.bypassdev=0x1000:0x15"
+            if "video=efifb:off" not in arg_str:
+                arg_str += " video=efifb:off"
+            if "video=VGA-1:640x480-32@60me" not in arg_str:
+                arg_str += " video=VGA-1:640x480-32@60me"
         else:
             arg_str += " crashkernel=512M"
 
-- 
2.23.0

