From e8ce5155f21af58e119c61e10895fcb5d8c21995 Mon Sep 17 00:00:00 2001
From: t_feng <fengtao40@huawei.com>
Date: Thu, 18 Jun 2020 17:00:13 +0800
Subject: [PATCH] add passwd policy

---
 data/interactive-defaults.ks | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/data/interactive-defaults.ks b/data/interactive-defaults.ks
index a906c8c..0177cf9 100644
--- a/data/interactive-defaults.ks
+++ b/data/interactive-defaults.ks
@@ -4,9 +4,9 @@ firstboot --enable
 
 %anaconda
 # Default password policies
-pwpolicy root --notstrict --minlen=6 --minquality=1 --nochanges --notempty
-pwpolicy user --notstrict --minlen=6 --minquality=1 --nochanges --emptyok
-pwpolicy luks --notstrict --minlen=6 --minquality=1 --nochanges --notempty
+pwpolicy root --notstrict --minlen=8 --minquality=1 --nochanges --notempty
+pwpolicy user --notstrict --minlen=8 --minquality=1 --nochanges --emptyok
+pwpolicy luks --notstrict --minlen=8 --minquality=1 --nochanges --notempty
 # NOTE: This applies only to *fully* interactive installations, partial kickstart
 #       installations use defaults specified in pyanaconda/pwpolicy.py.
 #       Automated kickstart installs simply ignore the password policy as the policy
-- 
2.23.0

