diff -Nur anaconda-19.31.123-orig/pyanaconda/anaconda_logging.py anaconda-19.31.123/pyanaconda/anaconda_logging.py
--- anaconda-19.31.123-orig/pyanaconda/anaconda_logging.py  2017-10-12 21:46:46.936000000 -0400
+++ anaconda-19.31.123/pyanaconda/anaconda_logging.py       2017-10-12 21:49:15.310000000 -0400
@@ -28,4 +28,5 @@
 import sys
 import warnings
+import time

 from pyanaconda.flags import flags
@@ -187,6 +187,7 @@
     def __init__(self):
         self.loglevel = DEFAULT_LEVEL
         self.remote_syslog = None
+        logging.Formatter.converter = time.gmtime
         # Rename the loglevels so they are the same as in syslog.
         logging.addLevelName(logging.CRITICAL, "CRT")
         logging.addLevelName(logging.ERROR, "ERR")
