diff -uNrp a/pyanaconda/core/util.py b/pyanaconda/core/util.py
--- a/pyanaconda/core/util.py	2018-10-09 03:00:59.000000000 +0800
+++ b/pyanaconda/core/util.py	2019-09-26 11:09:49.580000000 +0800
@@ -1354,7 +1354,6 @@ def collect(module_pattern, path, pred):
         module_path = None
 
         try:
-            imp.acquire_lock()
             (fo, module_path, module_flags) = imp.find_module(mod_name, [path])
             module = sys.modules.get(module_pattern % mod_name)
 
@@ -1427,8 +1426,6 @@ def collect(module_pattern, path, pred):
             log.error("Failed to import module %s from path %s in collect: %s", mod_name, module_path, imperr)
             continue
         finally:
-            imp.release_lock()
-
             if mod_info and mod_info[0]:  # pylint: disable=unsubscriptable-object
                 mod_info[0].close()  # pylint: disable=unsubscriptable-object
 
