From a5c305939ac0261c428da982891be2e753b54b4e Mon Sep 17 00:00:00 2001
From: fengtao <fengtao40@huawei.com>
Date: Sat, 11 Jan 2020 17:23:48 +0800
Subject: [PATCH] add boot options for raid 3408

---
 pyanaconda/bootloader.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyanaconda/bootloader.py b/pyanaconda/bootloader.py
index e0e93d5..4f96c99 100644
--- a/pyanaconda/bootloader.py
+++ b/pyanaconda/bootloader.py
@@ -1537,6 +1537,7 @@ class GRUB2(GRUB):
         if blivet.arch.is_aarch64():
             log.info("check boot args:%s",arg_str)
             arg_str=re.sub("console=[a-zA-Z0-9,]*","",arg_str)
+            arg_str+=" smmu.bypassdev=0x1000:0x17 smmu.bypassdev=0x1000:0x15"
         else:
             arg_str+=" quiet"
 
-- 
1.8.3.1

