diff -uNrp a/pyanaconda/ui/gui/hubs/__init__.py b/pyanaconda/ui/gui/hubs/__init__.py
--- a/pyanaconda/ui/gui/hubs/__init__.py	2018-07-25 21:58:20.000000000 +0800
+++ b/pyanaconda/ui/gui/hubs/__init__.py	2019-08-12 21:24:15.856000000 +0800
@@ -148,7 +148,7 @@ class Hub(GUIObject, common.Hub):
                 # From here on, this Spoke will always exist.
                 spoke = spokeClass(self.data, self.storage, self.payload, self.instclass)
                 spoke.window.set_beta(self.window.get_beta())
-                spoke.window.set_property("distribution", distributionText().upper())
+                spoke.window.set_property("distribution", distributionText())
 
                 # If a spoke is not showable, it is unreachable in the UI.  We
                 # might as well get rid of it.
diff -uNrp a/pyanaconda/ui/gui/__init__.py b/pyanaconda/ui/gui/__init__.py
--- a/pyanaconda/ui/gui/__init__.py	2018-06-08 05:31:59.000000000 +0800
+++ b/pyanaconda/ui/gui/__init__.py	2019-08-12 21:22:19.776000000 +0800
@@ -829,7 +829,7 @@ class GraphicalUserInterface(UserInterfa
             self._currentAction.refresh()
 
             self._currentAction.window.set_beta(not self._isFinal)
-            self._currentAction.window.set_property("distribution", self._distributionText().upper())
+            self._currentAction.window.set_property("distribution", self._distributionText())
 
             # Set some program-wide settings.
             settings = Gtk.Settings.get_default()
@@ -995,7 +995,7 @@ class GraphicalUserInterface(UserInterfa
 
         nextAction.initialize()
         nextAction.window.set_beta(self._currentAction.window.get_beta())
-        nextAction.window.set_property("distribution", self._distributionText().upper())
+        nextAction.window.set_property("distribution", self._distributionText())
 
         if not nextAction.showable:
             self._currentAction.window.hide()
diff -uNrp a/pyanaconda/ui/gui/spokes/welcome.py b/pyanaconda/ui/gui/spokes/welcome.py
--- a/pyanaconda/ui/gui/spokes/welcome.py	2018-10-09 03:00:59.000000000 +0800
+++ b/pyanaconda/ui/gui/spokes/welcome.py	2019-08-12 21:23:46.248000000 +0800
@@ -238,7 +238,7 @@ class WelcomeLanguageSpoke(LangLocaleHan
         welcomeLabel = self.builder.get_object("welcomeLabel")
 
         welcomeLabel.set_text(_("WELCOME TO %(name)s %(version)s.") %
-                {"name" : productName.upper(), "version" : productVersion})         # pylint: disable=no-member
+                {"name" : productName, "version" : productVersion})         # pylint: disable=no-member
 
         # Retranslate the language (filtering) entry's placeholder text
         languageEntry = self.builder.get_object("languageEntry")
