From 2cae8055f3d38e071969039ae96ae4696bf5832d Mon Sep 17 00:00:00 2001
From: root <root@localhost.localdomain>
Date: Thu, 12 Dec 2019 04:21:48 -0500
Subject: [PATCH] add quiet cmdline args for x86

---
 pyanaconda/bootloader.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/pyanaconda/bootloader.py b/pyanaconda/bootloader.py
index 4d69e31..e0e93d5 100644
--- a/pyanaconda/bootloader.py
+++ b/pyanaconda/bootloader.py
@@ -1537,6 +1537,8 @@ class GRUB2(GRUB):
         if blivet.arch.is_aarch64():
             log.info("check boot args:%s",arg_str)
             arg_str=re.sub("console=[a-zA-Z0-9,]*","",arg_str)
+        else:
+            arg_str+=" quiet"
 
         log.info("bootloader.py: used boot args: %s ", arg_str)
         defaults.write("GRUB_CMDLINE_LINUX=\"%s\"\n" % arg_str)
-- 
2.19.1

