--- anaconda-29.24.7/pyanaconda/users.py	
+++ anaconda-29.24.7/pyanaconda/users.py
@@ -397,7 +397,7 @@
         # Reset sp_lstchg to an empty string. On systems with no rtc, this
         # field can be set to 0, which has a special meaning that the password
         # must be reset on the next login.
-        util.execWithRedirect("chage", ["-R", root, "-d", "", username])
+        #util.execWithRedirect("chage", ["-R", root, "-d", "", username])
 
     def setRootPassword(self, password, isCrypted=False, isLocked=False, algo=None, root="/"):
         return self.setUserPassword("root", password, isCrypted, isLocked, algo, root)

