diff -Nur anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/root_password.py anaconda-29.24.7/pyanaconda/ui/gui/spokes/root_password.py
--- anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/root_password.py	2019-01-30 20:34:33.200096820 -0500
+++ anaconda-29.24.7/pyanaconda/ui/gui/spokes/root_password.py	2019-01-31 01:49:17.967096820 -0500
@@ -229,8 +229,7 @@
                 self.show_warning_message(error_message)
             else:
                 # add suffix for the click twice logic
-                self.show_warning_message("{} {}".format(error_message,
-                                                         _(constants.PASSWORD_DONE_TWICE)))
+                self.show_warning_message(error_message)
 
         # check if the spoke can be exited after the latest round of checks
         self._check_spoke_exit_conditions(unwaivable_check_failed)
diff -Nur anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/user.py anaconda-29.24.7/pyanaconda/ui/gui/spokes/user.py
--- anaconda-29.24.7.old/pyanaconda/ui/gui/spokes/user.py	2019-01-30 20:34:33.200096820 -0500
+++ anaconda-29.24.7/pyanaconda/ui/gui/spokes/user.py	2019-01-31 01:48:23.643096820 -0500
@@ -593,8 +593,7 @@
                 self.show_warning_message(error_message)
             else:
                 # add suffix for the click twice logic
-                self.show_warning_message("{} {}".format(error_message,
-                                                         _(constants.PASSWORD_DONE_TWICE)))
+                self.show_warning_message(error_message)
 
         # check if the spoke can be exited after the latest round of checks
         self._check_spoke_exit_conditions(unwaivable_check_failed)
