diff -Nur anaconda-29.24.7-old/pyanaconda/input_checking.py anaconda-29.24.7/pyanaconda/input_checking.py
--- anaconda-29.24.7-old/pyanaconda/input_checking.py	2018-06-07 17:31:59.000000000 -0400
+++ anaconda-29.24.7/pyanaconda/input_checking.py	2019-02-25 07:31:26.287338526 -0500
@@ -472,6 +472,10 @@
         else:
             pw_score = 4
             status_text = _(constants.SecretStatus.STRONG.value)
+        
+        #disable set password without confirnation
+        if not error_message and not check_request.password_confirmation:
+            error_message = _(constants.SECRET_CONFIRM_ERROR_GUI[check_request.secret_type])
 
         # the policy influences the overall success of the check
         # - score 0 & strict == True -> success = False
