--- a/pyanaconda/bootloader.py	2019-05-23 19:35:56.260000000 +0800
+++ b/pyanaconda/bootloader.py	2019-05-23 21:37:08.028000000 +0800
@@ -1531,8 +1531,15 @@
         # this is going to cause problems for systems containing multiple
         # linux installations or even multiple boot entries with different
         # boot arguments
-        log.info("bootloader.py: used boot args: %s ", self.boot_args)
-        defaults.write("GRUB_CMDLINE_LINUX=\"%s\"\n" % self.boot_args)
+
+        #on aarch64 do not set console=xxx
+        if blivet.arch.is_aarch64():
+            arg_str = "%s" % self.boot_args
+            log.info("check boot args:%s",arg_str)
+            arg_str=re.sub("console=[a-zA-Z0-9,]*","",arg_str)
+
+        log.info("bootloader.py: used boot args: %s ", arg_str)
+        defaults.write("GRUB_CMDLINE_LINUX=\"%s\"\n" % arg_str)
         defaults.write("GRUB_DISABLE_RECOVERY=\"true\"\n")
         #defaults.write("GRUB_THEME=\"/boot/grub2/themes/system/theme.txt\"\n")
         defaults.close()
