diff -Nur anaconda-29.24.7-old/pyanaconda/bootloader.py anaconda-29.24.7/pyanaconda/bootloader.py
--- anaconda-29.24.7-old/pyanaconda/bootloader.py	2019-07-09 22:14:24.800000000 -0400
+++ anaconda-29.24.7/pyanaconda/bootloader.py	2019-07-09 22:15:17.936000000 -0400
@@ -1533,8 +1533,8 @@
         # boot arguments
 
         #on aarch64 do not set console=xxx
+        arg_str = "%s" % self.boot_args
         if blivet.arch.is_aarch64():
-            arg_str = "%s" % self.boot_args
             log.info("check boot args:%s",arg_str)
             arg_str=re.sub("console=[a-zA-Z0-9,]*","",arg_str)
 
