From 22e65d85d4e49ba25cc8f65eb9f74c899ece47ff Mon Sep 17 00:00:00 2001
From: luochunsheng <luochunsheng@huawei.com>
Date: Tue, 9 Apr 2019 17:14:23 +0800
Subject: [PATCH] Subject: [PATCH] set right eula location

---
 pyanaconda/installclasses/rhel.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/installclasses/rhel.py b/pyanaconda/installclasses/rhel.py
index 40de99f..e31f909 100644
--- a/pyanaconda/installclasses/rhel.py
+++ b/pyanaconda/installclasses/rhel.py
@@ -48,7 +48,7 @@ class RHELBaseInstallClass(BaseInstallClass):
     help_placeholder_with_links = "rhel_help_placeholder.xml"
     help_placeholder_plain_text = "rhel_help_placeholder.txt"
 
-    eula_path="/usr/share/redhat-release/EULA"
+    eula_path="/usr/share/openEuler-release/EULA"
 
     mirrors_available = False
 
-- 
1.8.3.1

