diff -Nur anaconda-29.24.7.old/pyanaconda/core/constants.py anaconda-29.24.7/pyanaconda/core/constants.py
--- anaconda-29.24.7.old/pyanaconda/core/constants.py	2019-01-30 20:34:33.221096820 -0500
+++ anaconda-29.24.7/pyanaconda/core/constants.py	2019-01-31 03:37:25.427096820 -0500
@@ -156,7 +156,7 @@
     PASSPHRASE = "passphrase"
 
 # Password validation
-SECRET_MIN_LEN = 6
+SECRET_MIN_LEN = 8 
 SECRET_EMPTY_ERROR = {
     SecretType.PASSWORD : N_("The password is empty."),
     SecretType.PASSPHRASE : N_("The passphrase is empty.")
