From 66f30776cdb79ccb26be01ad9cd2e1712fc0170c Mon Sep 17 00:00:00 2001
From: fengtao <fengtao40@huawei.com>
Date: Mon, 9 Mar 2020 21:19:07 +0800
Subject: [PATCH] add boot options for dummy

---
 pyanaconda/bootloader.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/bootloader.py b/pyanaconda/bootloader.py
index 0fe5dd2..f539454 100644
--- a/pyanaconda/bootloader.py
+++ b/pyanaconda/bootloader.py
@@ -1537,7 +1537,7 @@ class GRUB2(GRUB):
         if blivet.arch.is_aarch64():
             log.info("check boot args:%s",arg_str)
             arg_str=re.sub("console=[a-zA-Z0-9,]*","",arg_str)
-            arg_str+=" smmu.bypassdev=0x1000:0x17 smmu.bypassdev=0x1000:0x15 crashkernel=1024M,high"
+            arg_str+=" smmu.bypassdev=0x1000:0x17 smmu.bypassdev=0x1000:0x15 crashkernel=1024M,high video=efifb:off video=VGA-1:640x480-32@60me"
         else:
             arg_str+=" quiet crashkernel=512M"
 
-- 
1.8.3.1

