From aa21df647d18390124379026dbaf99741d336875 Mon Sep 17 00:00:00 2001
From: t_feng <fengtao40@huawei.com>
Date: Fri, 19 Jun 2020 10:43:24 +0800
Subject: [PATCH] modify arguments parsing

---
 pyanaconda/argument_parsing.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/argument_parsing.py b/pyanaconda/argument_parsing.py
index 6b2f40f..ce29587 100644
--- a/pyanaconda/argument_parsing.py
+++ b/pyanaconda/argument_parsing.py
@@ -592,7 +592,7 @@ def getArgumentParser(version_string, boot_cmdline=None):
 
     # some defaults change based on cmdline flags
     if boot_cmdline is not None:
-        if "console" in boot_cmdline:
+        if "console" in boot_cmdline and "inst.text" in boot_cmdline:
             ap.set_defaults(display_mode=DisplayModes.TUI)
 
     return ap
-- 
2.23.0

